tests/examples/misc_examples.py

```diff
@@ -41,6 +41,8 @@
 
 MOCK_LOGAPI_URL = 'https://mydummyurl.com/management/logs'
 
+MOCK_LOGSETAPI_URL = 'https://mydummyurl.com/management/logsets'
```

Comment:
I think we can start removing mock urls from this file and declaring them as a constant in UT file itself. What do you think?